lunar/distribution.yaml

```diff
@@ -1063,6 +1063,21 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: lunar-devel
     status: maintained
+  pid:
+    doc:
+      type: git
+      url: https://bitbucket.org/AndyZe/pid.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
yeah it looks like it's a copy paste from the [kinetic entry](https://github.com/ros/rosdistro/blob/master/kinetic/distribution.yaml#L4544-L4558).
I think extending the test for this is a good idea! Is there any corner case where the release entry is not of the form `release/<ROSDITRO>/XX` ?